client/index.html

```diff
@@ -4,9 +4,27 @@
 		<meta charset="UTF-8" />
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>Document</title>
+		<title>Home</title>
 		<script defer src="./assets/bundle.js"></script>
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
 		<link rel="stylesheet" href="assets/css/style.css" />
 	</head>
-	<body></body>
+	<body>
+		<nav>
+			<h4 id="h4" class="navbar-grid-item">JOURNEL</h4>
+			<div class="navbar-grid-item" id='navbar-grid'>
+				<h5><a class="navbar-anchors" href="#">Anime</a></h5>
```

Comment:
class here is superfluous. can select with `nav div h5 a`

Reply:
I attempted this and the browser did not like that:
![Screenshot 2021-07-05 at 19 51 19](https://user-images.githubusercontent.com/78097320/124509838-820db200-ddca-11eb-8467-627b9ab52e85.png)